### DISCUSSION TITLE
Can I perform multiple incremental with graphrag (with version above 4.0)?

### DISCUSSION THREAD

**Discussion by MinzhiHuang** [microsoft/graphrag#1453] [Q&A]
Can I perform multiple incremental updates separately? I tried, but seems like the there is no change for the create_final_entities.parquet after my second time increment.  I kept the                                                                           update_index_storage:
    type: file # or blob
    base_dir: "update_output"  in my setting document.

**Comment by JoedNgangmeni** [comment]
I have a similar question. Can we update the knowledge base without having to rebuild the entire index?

**Comment by amitt0488** [comment]
I also have same question
